frontend/components/Dashboard/MyFeed/MyFeed.tsx

```diff
@@ -1,13 +1,11 @@
 import React from 'react'
-
-import PostCard from '../PostCard'
+import { Post } from '../../../generated/graphql'
+import MyFeedPostCard from '../PostCard/MyFeedPostCard'
```

Comment:
All of these references to `MyFeedPostCard` are 100% updated in the PR branched off of this one right?